docs/core/tools/dotnet-migrate.md

```diff
@@ -15,24 +15,27 @@ ms.date: 08/14/2017
 
 ## Synopsis
 
-`dotnet migrate [<SOLUTION_FILE|PROJECT_DIR>] [-t|--template-file] [-v|--sdk-package-version] [-x|--xproj-file] [-s|--skip-project-references] [-r|--report-file] [--format-report-file-json] [--skip-backup] [-h|--help]`
+```
```

Comment:
Should this be ```console?

Reply:
I haven't been using anything for the CLI commands. I talked to Curt and they will create a dotnetcli for us, but it'll take some time.